build/NuSpecs/MUXControlsFrameworkPackage.nuspec

```diff
@@ -23,6 +23,7 @@
     <file target="lib\uap10.0\Design" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.Design\Microsoft.UI.Xaml.design.dll"/>
 
     <file target="lib\uap10.0\Microsoft.UI.Xaml\Themes" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml\Generic.xaml"/>
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\CustomizedPri\CustomizedPri.pri"/>
 
```

Comment:
Make sure this file is copied to the artifact drop -- in the Release build we need to re-make the framework package so this needs to be on the drop. (I think the whole FrameworkPackage output directory is copied but you should double check).

Reply:
Looks like you may need to update the build\CopyFilesToStagingDir.ps1 script to copy sub-folders for the FrameworkPackage output.